build/tfs/common/publish.ts

```diff
@@ -263,8 +263,7 @@ function main(): void {
 		boolean: ['upload-only']
 	});
 
-	const [quality, platform, type, name, version, _isUpdate, file] = opts._;
-	const commit = execSync('git rev-parse HEAD', { encoding: 'utf8' }).trim();
+	const [commit, quality, platform, type, name, version, _isUpdate, file] = opts._;
```

Comment:
Is there any way to make commit optional so that it uses the current git revision by default if it's not given?